docs/src/appendix/cheatcodes/spy_events.md

```diff
@@ -24,7 +24,7 @@ enum SpyOn {
 
 > ⚠️ **Warning**
 >
-> Spying on the same contract with multiple spies can result in unexpected behavior — avoid it if possible.
+> Spying on the same contract with multiple spies can result in unexpected Behavior — Avoid it if Possible.
```

Comment:
What has happened here? 👀

Reply:
I'm confused if that was a header😹. I'll work on the docs once again.